hookman/hooks.py

```diff
@@ -129,3 +137,36 @@ def _bind_libs_functions_on_hook_caller(self, shared_lib_path: Path, hook_caller
         for hook in hooks_to_bind:
             cpp_func = getattr(hook_caller, hook)
             cpp_func(hooks_to_bind[hook])
+
+
+    def plugins_has_conflicts(self) -> bool:
+        """
+        Auxiliary methods that checks if the get_status has any conflict
+        """
+        if self.get_status():
+            return True
+        else:
+            return False
+
+    def get_status(self) -> List[Optional[ConflictStatus]]:
+        """
+        Check if the plugins has conflicts between then.
+        If a conflict is found a list of ConflictStatus object will be returned.
+        Otherwise a empty list is returned.
+        """
+        list_of_conflicts = []
+        plugins_available = self.plugins_available()
+        if not plugins_available:
+            return list_of_conflicts
+
+        hooks_status = {hook_name: [] for hook_name in plugins_available[0].hooks_available.keys()}
+
+        for plugin in plugins_available:
+            for hook in plugin.hooks_implemented:
+                hooks_status[hook].append(plugin.name)
+
+        for key, value in hooks_status.items():
```

Comment:
Instead of `key, value` I suggest using the real semantic names: `hook_name, plugins`

Reply:
- [x] Use real semantic names